operators/pkg/tenant-controller/tenant_controller.go

```diff
@@ -396,9 +413,49 @@ func (r *TenantReconciler) createOrUpdateClusterResources(ctx context.Context, t
 		retErr = err
 	}
 	klog.Infof("Allow network policy for tenant %s %s", tn.Name, npAOpRes)
+
+	retErr = r.createOrUpdateTnPersonalNFSVolume(ctx, tn, nsName)
```

Comment:
To keep the previous logic, you should not override `retErr` in case it is nil.

Reply:
Ok